### ISSUE TITLE
javascript:DuplicatePropertyName false positive on binding class properties

### ISSUE THREAD

**Issue by lukeapage** [SonarSource/SonarJS#1279]
**I want to report a bug.**

**SonarJS version:**  5.0.0.6962
**SonarQube version:** 7.6

**Rule key:** javascript:DuplicatePropertyName

**Reproducer** 
```
class A {
    myFunc = this.myFunc.bind(this);
    myFunc() {
    }
}
```

**Expected behavior**

This is a established pattern to bind handlers (its the same as if I bound my functions in the cnstructor, its just using class properties is less code). Therefore it should not error if Sonar detects the "duplicate" is just a rebinding of the method.

**Comment by lukeapage** [comment]
This is the most annoying as I have about 120 bugs reported which are this, which would need suppressing.

**Comment by yassin-kammoun-sonarsource** [comment]
Fixed with #2091.
